### PR TITLE
Switch to Pytest

### PR DESCRIPTION
**Pull request by mikedh**
This is an architectural change, so feel free to unilaterally close without debate if it's not something you want haha. 

This PR moves tests out of `collada/tests`, and removes the `__main__` method to run tests. It now runs tests using `pytest tests` rather than `python collada`, which uses the installed version of collada which should catch the issues like in #92. The downside is testing locally is less isolated, upside is travis should catch install issues. Also pytest is pretty nice, I'm a fan of their report style. 

I also changed it so `setup.py` and `collada/__init__` both pull the version from `collada/version.py`, as previously the value was stored in multiple places and they weren't all the same.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,10 +1,21 @@
...
 import sys
 from setuptools import find_packages, setup
 
+# load __version__ without importing anything
+version_file = os.path.join(
+    os.path.dirname(__file__),
+    'collada/version.py')
+with open(version_file, 'r') as f:
+    # use eval to get a clean string of version from file
+    __version__ = eval(f.read().strip().split('=')[-1])
```
Comment (paulmelnikow) [inline setup.py]: 👍

File: tests/test_geometry.py [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+
+import collada
+from collada.util import unittest
+from collada.xmlutil import etree
+
+fromstring = etree.fromstring
+tostring = etree.tostring
+
+
+class TestGeometry(unittest.TestCase):
```
Comment (paulmelnikow) [inline tests/test_geometry.py]: Is it possible to avoid rewriting this file in the diff, maybe by moving some of the reformatting to a separate PR?

**Comment by jterrace** [comment]
Sorry, I know you probably spent a fair amount of time on this, but I really don't want to merge. I prefer using the built-in unittest, even with its flaws. I don't want to adopt a different framework at this time, and I really prefer being able to run the tests after the module is installed.
